tutor

```diff
@@ -0,0 +1,995 @@
+     używać Vima jako edytora ogólnego przeznaczenia.
+
+     Czas potrzebny na ukończenie tutoriala to 25 do 30 minut i zależy
+     od tego jak wiele czasu spędzisz na eksperymentowaniu.
+
+	 UWAGA:
+	 Polecenia wykonywane w czasie lekcji zmodyfikują tekst. Zrób
+	 wcześniej kopię tego pliku do ćwiczeń (jeśli zacząłeś komendą
+	 "vimtutor", to już pracujesz na kopii).
+
+	 Pamiętaj, że przewodnik ten został zaprojektowany do nauki poprzez
+	 ćwiczenia. Oznacza to, że musisz wykonywać polecenia, by nauczyć się ich
+	 prawidłowo. Jeśli będziesz jedynie czytał tekst, szybko zapomnisz wiele
+	 poleceń!
+
+     Teraz upewnij się, że nie masz wciśniętego Caps Locka i wciskaj  j
+     tak długo dopóki Lekcja 1.1. nie wypełni całkowicie ekranu.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+		  Lekcja 1.1.: PORUSZANIE SIĘ KURSOREM
+
+       ** By wykonać ruch kursorem, wciśnij h, j, k, l jak pokazano. **
+
+	       ^
+	       k		      Wskazówka:  h jest po lewej
+	  < h	  l >				  l jest po prawej
+	       j				  j wygląda jak strzałka w dół
+	       v
+  1. Poruszaj kursorem dopóki nie będziesz pewien, że pamiętasz polecenia.
+
+  2. Trzymaj  j  tak długo aż będzie się powtarzał.
+     Teraz wiesz jak dojść do następnej lekcji.
+
+  3. Używając strzałki w dół przejdź do następnej lekcji.
+
+Uwaga: Jeśli nie jesteś pewien czegoś co wpisałeś, wciśnij <ESC>, by wrócić do
+       trybu Normal. Wtedy powtórz polecenie.
+
+Uwaga: Klawisze kursora także powinny działać, ale używając  hjkl  będziesz
+       w stanie poruszać się o wiele szybciej, jak się tylko przyzwyczaisz.
+       Naprawdę!
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+		    Lekcja 1.2.: WYCHODZENIE Z VIM-a
+
+ !! UWAGA: Przed wykonaniem jakiegokolwiek polecenia przeczytaj całą lekcję !!
+
+  1. Wciśnij <ESC> (aby upewnić się, że jesteś w trybie Normal).
+  2. Wpisz:			:q!<ENTER>.
+     To spowoduje wyjście z edytora PORZUCAJĄC wszelkie zmiany, jakie
+     zdążyłeś zrobić. Jeśli chcesz zapamiętać zmiany i wyjść,
+     wpisz:			:wq<ENTER>
+
+  3. Kiedy widzisz znak zachęty powłoki wpisz komendę, żeby wrócić
+     do tutoriala. Czyli:	vimtutor<ENTER>
+
+  4. Jeśli chcesz zapamiętać polecenia, wykonaj kroki 1. do 3., aby
+     wyjść i wrócić do edytora.
+
+UWAGA: :q!<ENTER> porzuca wszelkie zmiany jakie zrobiłeś. W następnych
+       lekcjach dowiesz się jak je zapamiętywać.
+
+  5. Przenieś kursor do lekcji 1.3.
+
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+		 Lekcja 1.3.: EDYCJA TEKSTU - KASOWANIE
+
+	    ** Wciśnij  x  aby usunąć znak pod kursorem. **
+
+  1. Przenieś kursor do linii poniżej oznaczonej --->.
+
+  2. By poprawić błędy, naprowadź kursor na znak do usunięcia.
+
+  3. Wciśnij  x  aby usunąć niechciany znak.
+
+  4. Powtarzaj kroki 2. do 4. dopóki zdanie nie jest poprawne.
+
+---> Kkrowa prrzeskoczyła prrzez ksiiężycc.
```

Comment:
hmm... not completed

Reply:
Found that right file has been saved in different place.